scripts/zones/Bastok_Markets/npcs/Trick_Bones.lua

```diff
@@ -0,0 +1,46 @@
+-----------------------------------
+-- Area: Bastok Markets
+--  NPC: Trick Bones
+-- Harvest Festival Roaming Monster
+-----------------------------------
+local ID = require("scripts/zones/Bastok_Markets/IDs")
+require("scripts/globals/pathfind")
+require("scripts/globals/utils")
```

Comment:
do we have to also require the harvest festival library?

Reply:
No reason to, they are all global functions